packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -77,8 +77,11 @@
                     "valueQuantity": {{>DataType/Quantity.hbs quantity=../observationEntry.value}},
                 {{/if}}
             {{/with}}
-        {{else}}
+        {{else if observationEntry.value._}}
             "valueString":"{{{parseReferenceData observationEntry.value._}}}",
+        {{!-- in some cases text is here in the b64 field. Adding so no data loss but not sure if thats the way its supposed to be. also how it renders I do not know         --}}
+        {{else if observationEntry.value._b64}}
```

Comment:
I was seeing the actual core piece of the observation being lost in several observations with references. Turned out the b64 data was stored in _b64. This might need work on the MR gen side since b64 is nested in observation resource. 

Also not sure if we are putting info in both _ and _b64, or if the former was supposed to be migrated to the latter and we just forgot to

Reply:
was this still a thing even after the fix we pushed yesterday?